examples/c/src/phi3v.cpp

```diff
@@ -92,36 +103,49 @@ void C_API(const char* model_path) {
   CheckResult(OgaCreateTokenizerStreamFromProcessor(processor, &tokenizer_stream));
 
   while (true) {
-    std::string image_path;
-    std::cout << "Image Path (leave empty if no image):" << std::endl;
-    std::getline(std::cin, image_path);
+    std::string image_paths_str;
+    std::cout << "Image Path (comma separated; leave empty if no image):" << std::endl;
+    std::getline(std::cin, image_paths_str);
     OgaImages* images = nullptr;
-    if (image_path.empty()) {
+    std::vector<std::string> image_paths;
+    for (size_t start = 0, end = 0; end < image_paths_str.size(); start = end + 1) {
+      end = image_paths_str.find(',', start);
```

Comment:
Will the processor be affected by whitespace in the paths when using `.find()` and `.substr()` or is leading and trailing whitespace already stripped?

For example, whether the returned paths are `path_1`, ` path_2`, and ` path_3` (if a user passes the paths as `path_1, path_2, path_3` with a space after each comma) or whether the returned paths are `path_1`, `path_2`, and `path_3` (if a user passes the paths as `path_1,path_2,path_3` without a space after each comma), both cases should work.

Reply:
Whitespaces will cause a problem as it stands right now. I can modify to trim the strings after extracting the paths using the delimiter.